pyLibrary/queries/expressions.py

```diff
@@ -274,7 +274,7 @@ def to_python(self, not_null=False, boolean=False, many=False):
         return output
 
     def to_sql(self, schema, not_null=False, boolean=False):
-        cols = [c for cname, cs in schema.items() if startswith_field(cname, self.var) for c in cs]
+        cols = [c for cname, cs in schema.items() if startswith_field(cname, self.var.replace("\\.", ".")) for c in cs]
```

Comment:
This is not correct: I have not problem with using `.replace("\\.", ".")` for formatting, but using for comparing fields points to a deeper issue in the code.  I will merge this pull request with this file reverted.